models/experimental/functional_roberta/tests/test_perf_device_roberta.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from models.utility_functions import skip_for_wormhole_b0
+from models.perf.device_perf_utils import run_device_perf, check_device_perf, prep_device_perf_report
+
+
+@skip_for_wormhole_b0()
+@pytest.mark.models_device_performance_bare_metal
+@pytest.mark.models_performance_bare_metal
```

Comment:
Get rid of `models_performance_bare_metal`. This is a device test

Reply:
Both decorators were inadvertently added. I have now updated the test with `models_device_performance_bare_metal` only.
Thank you.